hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NodeExecutor.java

```diff
@@ -51,6 +57,11 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
         .setTaskId(taskInfo.getTaskId())
         .setState(TaskState.TASK_RUNNING)
         .setData(taskInfo.getData()).build());
+    timer = new Timer(true);
+    timedHealthCheck = new TimedHealthCheck(driver, task);
+    timer.scheduleAtFixedRate(timedHealthCheck,
```

Comment:
Again, TimedHealthCheck can be constructed on demand and doesn't need to be stored in a private member variable.